Plugins/NativeXr/Source/NativeXr.cpp

```diff
@@ -1459,6 +1459,49 @@ namespace Babylon
             XRFrame* m_frame{};
         };
 
+        class XRHand : public Napi::ObjectWrap<XRHand>
+        {
+            static constexpr auto JS_CLASS_NAME = "XRHand";
+
+        public:
+            static void Initialize(Napi::Env env)
+            {
+                Napi::HandleScope scope{env};
+
+                std::vector<Napi::ClassPropertyDescriptor<XRHand>> initList = {};
```

Comment:
Since you know the size up front that you'll need for the `std::vector`, use the constructor that takes a `size_t` parameter so it'll pre-allocate its internal buffer to match.

Also nit: you don't need to do `= {}`. You can just do `std::vector<...> initList{}` if you're using the default constructor.